mesh-worker-service/src/main/java/io/functionmesh/compute/rest/api/FunctionsImpl.java

```diff
@@ -69,17 +69,26 @@ private void validateRegisterFunctionRequestParams(String tenant, String namespa
         if (functionConfig == null) {
             throw new RestException(Response.Status.BAD_REQUEST, "Function config is not provided");
         }
+        if (uploadedJar && !(Boolean) worker().getWorkerConfig().getRuntimeCustomizerConfig().get("uploadEnabled")) {
```

Comment:
Should the code handle the situation that these configurations are not set and in which case the feature should be enabled by default?